sim/build.sbt

```diff
@@ -21,7 +21,8 @@ lazy val testchipip = project in file("target-rtl/firechip/testchipip") settings
 lazy val icenet     = project in file("target-rtl/firechip/icenet") settings commonSettings dependsOn (rocketchip, testchipip)
 
 lazy val mdf        = RootProject(file("barstools/mdf/scalalib"))
-lazy val barstools  = project in file("barstools/macros") settings commonSettings dependsOn (mdf, rocketchip)
+lazy val firrtl     = project in file("firrtl") settings commonSettings
```

Comment:
we should be using the firrtl installed with publishLocal right?

Reply:
That causes some issues when firrtl updates. You end up having to redo the publishLocal. But I guess we have to do it for rocket-chip anyway, so I'll get rid of it.

Reply:
👍